config/custom-environment-variables.js

```diff
@@ -58,6 +58,10 @@ module.exports = {
     privateKey: "PRIVATE_KEY",
   },
 
+  botToken: {
+    publicKey: "PUBLIC_KEY",
```

Comment:
Can we change the name here, can we be more specific
And there already a key present with the same name

Reply:
Made the change